docs/word/create-better-add-ins-for-word-with-office-open-xml.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Create better add-ins for Word with Office Open XML
 description: ''
-ms.date: 03/19/2019
+ms.date: 11/10/2019
```

Comment:
```suggestion
ms.date: 10/10/2019
```

Reply:
Lol that's the European in me trying to break out